test/ref.jsx

```diff
@@ -31,14 +31,23 @@ describe('ref', () => {
     });
 
     it('passes with legacy refs', () => {
-      assertPasses(validator, <div someRef={() => {}} />, 'someRef');
+      assertPasses(validator, <div someRef={(node) => {}} />, 'someRef'); // eslint-disable-line no-unused-vars
```

Comment:
I don't love disabling lint rules. In the two cases in this file, I couldn't really think of a reasonable way around it, though.